lib/index.js

```diff
@@ -0,0 +1,127 @@
+/*
+ * Bedrock mail module.
+ *
+ * Copyright (c) 2012-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+const Email = require('email-templates');
+const appRoot = require('app-root-path');
+const bedrock = require('bedrock');
+const nodemailer = require('nodemailer');
+const path = require('path');
+
+const BedrockError = bedrock.util.BedrockError;
+
+// load config defaults
+require('./config');
+
+const api = {};
+module.exports = api;
+
+// email client
+let client;
+
+const logger = bedrock.loggers.get('app').child('bedrock-mail');
```

Comment:
It is my understanding that our convention is to have the logger separated into its own file and pulled in. If we want to break that pattern here, then that's fine.

Reply:
That should certainly be done to share the logger when multiple files are logging.  But in this case, it's just one main file that does logging.  I figured we didn't need to bother for the simple one file use case.

Reply:
I made an issue: https://github.com/digitalbazaar/bedrock-mail/issues/5